cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -1397,6 +1404,37 @@ public void bindDropletToApp(UUID dropletGuid, UUID applicationGuid) {
                 .block();
     }
 
+    @Override
+    public DropletInfo getCurrentDropletForApplication(UUID applicationGuid) {
+        GetApplicationCurrentDropletResponse getApplicationCurrentDropletResponse = delegate.applicationsV3()
+                                                                                            .getCurrentDroplet(GetApplicationCurrentDropletRequest.builder()
+                                                                                                                                                  .applicationId(applicationGuid.toString())
+                                                                                                                                                  .build())
+                                                                                            .block();
+        String packageUrl = getApplicationCurrentDropletResponse.getLinks()
```

Comment:
Is it possible to get package guid with another method without custom parse of href link?

Reply:
I think it is possible to get all created builds for an application, then filter by droplet id, get the build and get the package id through the build. In my opinion, this is at least one additional call to the controller and requires more processing.

Reply:
If href link ends with slash `/` then parse logic will fail. We have to think can we return null object and create new package if we couldn't parse existing one.